c++/src/capnp/layout.c++

```diff
@@ -701,7 +702,7 @@ struct WireHelpers {
             WirePointer* elementTag = reinterpret_cast<WirePointer*>(ptr);
 
             KJ_ASSERT(elementTag->kind() == WirePointer::STRUCT,
-                  "Don't know how to handle non-STRUCT inline composite.");
+                  "Schema mismatch: Don't know how to handle non-STRUCT inline composite.");
```

Comment:
This one isn't a schema mismatch. In fact `zeroObject()` doesn't care about schemas.